searchlib/src/tests/attribute/stringattribute/stringattribute_test.cpp

```diff
@@ -408,4 +409,83 @@ TEST("testSingleValue")
     }
 }
 
+TEST("test uncased match") {
+    QueryTermUCS4 xyz("xyz", QueryTermSimple::Type::WORD);
+    StringSearchHelper helper(xyz, false);
+    EXPECT_FALSE(helper.isCased());
+    EXPECT_FALSE(helper.isPrefix());
+    EXPECT_FALSE(helper.isRegex());
+    EXPECT_FALSE(helper.isMatch("axyz"));
+    EXPECT_FALSE(helper.isMatch("xyza"));
+    EXPECT_TRUE(helper.isMatch("xyz"));
+    EXPECT_TRUE(helper.isMatch("XyZ"));
```

Comment:
Consider adding a test case where the input string is shorter than the query term. Applies for all tests.

Reply:
Fixed